runtime/calamari/src/lib.rs

```diff
@@ -228,7 +213,8 @@ parameter_types! {
 }
 
 impl pallet_transaction_payment::Config for Runtime {
-	type OnChargeTransaction = pallet_transaction_payment::CurrencyAdapter<Balances, ()>;
+	type OnChargeTransaction =
+		pallet_transaction_payment::CurrencyAdapter<Balances, DealWithFees<Self>>;
```

Comment:
@stechu @Dengjianping 
This is the configuration which sends part of the transaction fees to the treasury and another part to the block author.

Currently the other 2 ways the treasury accumulates funds is through slashing from canceling proposals or slashing from canceling treasury proposals.

Reply:
This `DealWithFees` is from polkadot, we might to implement our own one. Perhaps we need to discuss how distribute it.

```
for fees, 80% to treasury, 20% to author
```
https://github.com/paritytech/polkadot/blob/master/runtime/common/src/impls.rs#L45

Reply:
We should put 100% fees to treasury for now. There's no author at all.